taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -150,15 +152,11 @@ def enable_webrender(config, tasks):
     for task in tasks:
         if not task.pop("web-render-only", False):
             newtask = copy.deepcopy(task)
+            task["name"].replace("-qr", "")
```

Comment:
This will also modify the original task when I think you only want to modify `newtask`.